doc/release/upcoming_changes/21029.expired.rst

```diff
@@ -0,0 +1,9 @@
+Multidimensional indexing with non-tuple values is no longer permitted
+----------------------------------------------------------------------
+
+Multidimensional indexing with anything but a tuple was
+deprecated in NumPy 1.15. Code such as ``ind = [slice(None), 0]``,
+``arr[[slice(None), 0]]`` should be changed to ``arr[tuple(ind)]``. This is
```

Comment:
Nitpick, but the code must be changed.  (interesting that in Python 3.11 you will also be able to write `arr[*[1, 2]]`, I think.  But not for here.